pkg/components/prometheus-operator/template.go

```diff
@@ -55,6 +57,21 @@ grafana:
       searchNamespace: ALL
   rbac:
     pspUseAppArmor: false
+
+  ingress:
+    enabled: {{ .ExposeGrafana }}
```

Comment:
Maybe just `{{printf "%t" .GrafanaIngressHost != ""}}` instead of separate filed?